src/Heading/index.js

```diff
@@ -31,6 +31,33 @@ const Anchor = ({ id }) => (
   />
 )
 
+const generateCss = (isHeading1, isHeading2) => {
+  if (isHeading1) {
+    return `& + p {
+      margin-top: var(--spectrum-global-dimension-static-size-300) !important;
+      font-size: var(--spectrum-global-dimension-static-size-225);
+    }`
+  } else if (isHeading2) {
+    return `&:nth-last-child(2) a:last-of-type {
+      opacity: 0;
+      transition: opacity var(--spectrum-global-animation-duration-100) ease-in-out;
+    }
+
+    &:nth-last-child(2):hover a:last-of-type {
+      opacity: 1;
+    }`
+  } else {
+    return `& a:last-of-type {
+      opacity: 0;
```

Comment:
same here. a test maybe? unless the stories cover it somehow.

Reply:
Good idea to add more tests. I've pushed ones that will check for the CSS classes. For the opacity business with the `#` at the end of the Heading I've got it covered in the stories. It would be great to have automated tests for this but I'm going to put that in the parking lot for now.